benchmarks/tests/test_solvers.py

```diff
@@ -132,11 +132,9 @@ def qubit_setup(size):
 
 
 @pytest.mark.nightly
-def test_mesolve(benchmark, model_solve, size, request):
-    # Group benchmark by operation, density and size.
-    group = request.node.callspec.id
-    group = "mesolve-" + group
-    benchmark.group = group
+def test_mesolve(benchmark, model_solve, size):
+
+    benchmark.group = "MeSolve"
```

Comment:
```suggestion
    benchmark.group = f"solvers:master-equation:{model_solve.lower()}"
```
As explained before the group name should represent a task -- in this case solve the master equation for the given model.

I don't know if this will require changing anything in the plotting code or not, but I think it is the right organization for the group names.

Reply:
Adding the model will involve changing the plotting functions, for it to remain the same something like `solvers:master-equation` would work. This is what I've done for now.
Should the model be needed, I can keep your suggestion for the group name and add what would be used for plot titles and file names to` benchmark.extra_info`.